config/nrfconnect/chip-module/Kconfig

```diff
@@ -47,9 +47,17 @@ config CHIP_OPENTHREAD_CONFIG
 	  This option can be used to replace Zephyr-supplied OpenThread
 	  configuration file with a custom one.
 
-config CHIP_LIB_SHELL
+config CHIP_ZEPHYR_SHELL
+	bool "Enable Zephyr-based CHIP shell"
+	default n
+	imply SHELL
+	help
+	  Add CHIP commands to the Zephyr shell.
+
+config CHIP_STANDALONE_SHELL
```

Comment:
is standalong shell still needed on nrfconnect? why we still need to keep the config of it?

Reply:
I think for now it's just a matter of backwards compatibility. Shell example still exists, can be run on nrfconnect and in the end doesn't offer the same set of commands as shell added temporarily by me, so someone may want to use it and I would rather remove it in case of removing whole shell example. Also I believe that shell added by me will be removed in the future, when chip-shell API will be exposed to chip src and usable in all examples, so we won't need my implementation anymore.